frontend/src/components/eMIB/ActionViewTask.jsx

```diff
@@ -14,14 +14,21 @@ const styles = {
   headings: {
     fontWeight: "bold"
   },
-  hr: {
-    margin: "16px 0 16px 0"
-  },
   editButton: {
     float: "right"
   },
   preWrap: {
     whiteSpace: "pre-wrap"
+  },
+  optionButtons: {
+    marginTop: 16,
+    paddingTop: 16,
+    borderTop: "1px solid rgba(0, 0, 0, 0.1)"
+  },
+  reasonsForAction: {
+    marginTop: 16,
+    paddingTop: 16,
+    borderTop: "1px solid rgba(0, 0, 0, 0.1)"
```

Comment:
Why do we have the exact same style here? Can we not have only one instead of a duplicate style?